wallet/walletdb/example_test.go

```diff
@@ -14,8 +14,8 @@ import (
 	"os"
 	"path/filepath"
 
-	_ "decred.org/dcrwallet/v3/wallet/internal/bdb"
 	"decred.org/dcrwallet/v3/errors"
+	_ "decred.org/dcrwallet/v3/wallet/internal/bdb"
```

Comment:
These imports got affected by me running:
```
gofmt -w -s $(find . -not -name "*.gen.go" -not -path "*/mock/*")
```
Not sure whether whether you'd want this change or not, leaving here for now (`go fmt ./...` by the way also changes similar imports on **master**).